dokan/dokan.c

```diff
@@ -570,44 +566,20 @@ DispatchCommon(PEVENT_CONTEXT EventContext, ULONG SizeOfEventInfo,
   return eventInfo;
 }
 
-PDOKAN_OPEN_INFO
-GetDokanOpenInfo(PEVENT_CONTEXT EventContext, PDOKAN_INSTANCE DokanInstance) {
-  PDOKAN_OPEN_INFO openInfo;
-  EnterCriticalSection(&DokanInstance->CriticalSection);
-
-  openInfo = (PDOKAN_OPEN_INFO)(UINT_PTR)EventContext->Context;
-  if (openInfo != NULL) {
-    openInfo->OpenCount++;
-    openInfo->EventContext = EventContext;
```

Comment:
FYI, I just updated my patch so that the per thread EventContext pointer is no longer stored in the shared DOKAN_OPEN_INFO struct. Instead it's now stored in the DOKAN_FILE_INFO struct which is per thread if I understand correctly.